### DISCUSSION TITLE
SD Error

### DISCUSSION THREAD

**Discussion by Ruggerio150** [sanni/cartreader#126] [Need help]
hello, I assembled the cardreader exactly according to the instructions. When switching on, the error message SD Error appears immediately. I tested 3 different SD cards. Samsung, sandisc ultra and sandisc extreme. I also checked the soldering points and checked the connection from the Arduino to the SD card adapter. Everything seems fine. I also formatted the cards with Fat32 and copied the contents of the SD folder into the root of the card. does anyone still have an idea what it could be?

many thanks

**Comment by sanni** [comment]
I have updated the SD library to the latest version in V6.1. Maybe it helps with SD compatibility.

**Reply by Ruggerio150** [reply]
Thank you very much. I found the error. The SD Card was damage. I buy a new and ist works Perfect. 👍
